CHANGELOG.md

```diff
@@ -1,6 +1,34 @@
 ## current master
 
-* update ohsome parent to version 2.3 (for building javadoc with JDK 11)
+
+## 1.1
+
+### Prominent Changes 
+
+* making filter2 parameter mandatory for /ratio requests and changing filter to be allowed to be null or empty (2dda1f8)
+* upgrading to ohsome filter 1.2 (9b061f5)
+* update of readme docs to give little intro and more usability examples (ae33581)
+
+### New Features
+
+* addition of parameter ‘clipGeometry’ for data-extraction endpoints (a0bf957)
+* restricting each parameter to be unique in the request (2183bb7)
+
+### Bug Fixes
+
+* adapting /groupBy/tag csv response to give empty result if no data matches given filters (ec4bacf)
+* Exception when no parameters were given (0691183)
+
+### Performance and Code Quality
+
+* modifying zero-fill usage (a8856c9)
```

Comment:
IMHO, this alone is not super informative. maybe it could be explained or otherwise left out from the changelog?

Reply:
Yeah I only found that one that fully fits in this category, as the other code quality and performance improvements were parts of a new feature or such and therefore that info is written in a different category.

I could rename this to "refactoring of diverse code parts, e.g. zero-fill usage (link), handling of clipping the geometries (link)"

Reply:
if there is only one point in the section (and the point is not even very major), then maybe it would be better to leave it away and put it under "other changes". I like the text you suggested :+1: